### ISSUE TITLE
UI when no results are available

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#898]
I just search a title for a book that we apparently don't have, the search returns no results (which is normal), but it's just a bit empty:

https://library-dev.bdrc.io/osearch/search?q=gsang%20can%20yig%20rigs

![Capture d’écran de 2024-07-23 14-44-55](https://github.com/user-attachments/assets/84793b5e-79f5-47aa-840a-c9a29edc1172)

I think we should have something a bit nicer, on the old website there was a little message:

![Capture d’écran de 2024-07-23 14-46-39](https://github.com/user-attachments/assets/3019de1a-1c7c-40b8-86e6-450b8721765f)

this message is a bit too verbose I think but at least it doesn't look like a bug happened.

It's quite interesting because I looked at other systems like https://gallica.bnf.fr/ or https://hollis.harvard.edu/primo-explore/search?vid=HVD2&sortby=rank&lang=en_US and they just return garbage results (a "or" on the syllables I think), which is an interesting way of saying "we don't have it"... I'm not sure I like it but it's a way at least.

What are your thoughts on that @roopeux ?

**Comment by berger-n** [comment]
looks a bit better now: https://library-dev.bdrc.io/osearch/search?q=dri%20med%20%27od%20zer%20test

![image](https://github.com/user-attachments/assets/83563935-28ca-42ff-bb99-a9c54a558450)

there's also the case of a server error:  http://library-dev.bdrc.io/osearch/search?q=dri%20med%20%27od%20zer&sortBy=publicationDate_desc
(sorting is not usable for now)

![image](https://github.com/user-attachments/assets/5d3c3b45-5328-40a0-9529-5b1fdf5b61ab)

**Comment by roopeux** [comment]
Yes it looks fine.  Later, we can add a link to a looser and fuzzier search with a light warning about quality.
